docs/docs/configuration/databases.mdx

```diff
@@ -1336,6 +1337,26 @@ starrocks://<User>:<Password>@<Host>:<Port>/<Catalog>.<Database>
 StarRocks maintains their Superset docuementation [here](https://docs.starrocks.io/docs/integrations/BI_integrations/Superset/).
 :::
 
+#### TDengine
+
+The recommended connector library for TDengine is
+[taospy](https://pypi.org/project/taospy/).
+You need to install the extras as well for this library.
+We recommend adding something like the following
+text to your requirements file:
+
+```
+taospy>=2.7.19
+taos-ws-py>=0.3.5
+```
```

Comment:
These should be defined in `pyproject.toml`. See #32052 for an example.

Reply:
done